Добавила разметку секции footer

Kate-Kompani opened the pull request:
Дивы буду добавлять при потребности уже когда приступим к css.
Пожалуйста проверьте путь к спрайту svg.  Не могу сообразить, как нашем случае это будет.

Kate-Kompani committed:
Добавила разметку секции footer

Дивы буду добавлять при потребности уже когда приступим к css.
Пожалуйста проверьте путь к спрайту svg.  Не могу сообразить, как нашем случае это будет.

Akim3351 reviewed (changes requested):
В лого: лучше Hell и EN - разделить на два класса, т.к. EN - это en, но прописанный апперкейсом в стилях. Фраза "адский английский" тоже пишется полностью строчными, а потом в стилях также добавляется апперкейс. Иначе будет срабатывать скример, читалки будут кричать. У меня в хедере тоже лого, но без ссылки, посмотри, как я сделал там.

В контактах: в ссылке телефона "href="tel:..." пробелы внутри номера не нужны, текст - тот номер, который прописан после закрытого тега СВГ.
Внутри лишек я бы "а" не ставил, они там незачем, но это некритично.

В самом низу на макете не значок копирайта, а "@" - видимо, так специально задумано)))

Kate-Kompani committed:
Внесла правки в секцию footer

1) Ссылки телефона и инстаграма, согласно тех заданию должны открываться в новой вкладке - исправила
2) Убрала капс  в логотипе
3) Добавила классы для дальнейшей стилизации
4) Инконка инстаграма будет добавляться в сss как фон.
5) Убрала пробелы в телефоне.
6) Копирайт оставила - по смыслу думаю, должен быть именно тот значек, а не @. Вынесу на обсуждение команды)

Kate-Kompani committed:
Добавила тег <footer>
